src/index.ts

```diff
@@ -14,6 +14,16 @@ import {
 	hasValidDleq
 } from './utils.js';
 import { CashuAuthMint, CashuAuthWallet, getBlindedAuthToken, getEncodedAuthToken } from './auth';
```

Comment:
Assuming the [modularized exports PR](https://github.com/cashubtc/cashu-ts/pull/297) is added first, we should not need to expose the NUT-11 utility functions to the public API here.

Reply:
Have removed this section now as modular exports have been merged.